source/includes/usage-examples/sample-read-app.c

```diff
@@ -0,0 +1,34 @@
+#include <bson/bson.h>
+#include <mongoc/mongoc.h>
+#include <stdio.h>
+
+int
+main (int argc, char *argv[])
+{
+    mongoc_client_t *client;
+    mongoc_collection_t *collection;
+    mongoc_cursor_t *results;
+    int64_t count;
+    const bson_t *doc;
+    bson_t *query, *command;
+    bson_t reply;
+    char *str;
+    bson_error_t error;
+
+    mongoc_init ();
+
+    client =
+        mongoc_client_new ("<connection string URI>");
+    collection = mongoc_client_get_collection (client, "<database name>", "collection name");
```

Comment:
S: If you are reducing horizontal scroll on the client line, also move the collection access to a new line